### PR TITLE
Fix updating page advancement on the first load

### LINKED ISSUE #1907: Changing tab to "document" creates a history entry for no reason
## Bug Description

On a saved story, the basic action of changing the inspector tab from "design" to "document" creates a history entry (which can be seen by the "save" button re-enabling).

Simply changing tabs should not edit the story.

## Expected Behaviour

Changing tabs should not edit the story.

## Steps to Reproduce

1. Switch to the design panel if you aren't already
1. Save the story and observe the "save" button being disabled (because the story hasn't changed since last save)
1. Switch tab to "document" and observe the "save" button being re-enabled (because the story somehow changed since last save)

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### LINKED ISSUE #4094: History: Opening an existing story indicates new changes right away
## Bug Description
When adding a new clean story without any changes, or opening an existing story, the Save Draft button is active, the Undo button is active, and trying to close the Story warns about unsaved changes. 

## Expected Behaviour
Opening a story without making changes should not indicate having changes.

## Steps to Reproduce
1. Create a new story
2. See the Save Draft and Undo button active.
3. Try to close the tab.
4. See the warning.

## Screenshots
new

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## To-do

- [x] Tests

## Testing Instructions
1. Add a new Story
2. Verify that "Save draft" button is not active without making any changes
3. Switch to the Document panel
4. Verify that "Save draft" button doesn't change to active.

---

Fixes #4094 
Fixes #1907

### CODE REVIEW

File: assets/src/edit-story/components/inspector/document/pageAdvancement.js [hunk 1]
```diff
@@ -79,9 +79,11 @@ function PageAdvancementPanel() {
     if (value !== newValue) {
       setDuration(newValue);
     }
-    updateStory({
-      properties: { defaultPageDuration: newValue },
-    });
+    if (defaultPageDuration !== newValue) {
```
Comment (swissspidy) [inline assets/src/edit-story/components/inspector/document/pageAdvancement.js:82]: What if I am changing the duration back from my current value to something that happens to be the default? Then I cannot do that?

Comment (miina) [reply assets/src/edit-story/components/inspector/document/pageAdvancement.js:82]: Hmm, but `defaultPageDuration` is a changing value, just the default value is `DEFAULT_PAGE_DURATION`, no?

If the user has changed the value meanwhile to something else, e.g, `10`, then the value of `defaultPageDuration` will also be `10`. And then when the user wants to change it back to `7`, that should be possible.

LMK if I'm missing something.

Comment (miina) [reply assets/src/edit-story/components/inspector/document/pageAdvancement.js:82]: Perhaps a more precise name for `DEFAULT_PAGE_DURATION` would actually be `DEFAULT_DEFAULT_PAGE_DURATION`.

Comment (swissspidy) [reply assets/src/edit-story/components/inspector/document/pageAdvancement.js:82]: Got it! Guess I confused it there :-)

**Review by barklund — APPROVED** [review]
Works as expected!

**Comment by miina** [comment]
There's an issue with Karma tests (#4364 ) which blocks merging this, however, sending to QA meanwhile already.

**Comment by csossi** [comment]
Verified in QA

**Review by spacedmonkey — APPROVED** [review]
LGTM

**Comment by miina** [comment]
Props @swissspidy for looking into the failing tests + config issue which in the end came out to be a test issue instead.

**Pull request merged**